website/docs/d/advanced_cluster.html.markdown

```diff
@@ -74,15 +75,25 @@ Specifies BI Connector for Atlas configuration.
 * `enabled` - Specifies whether or not BI Connector for Atlas is enabled on the cluster.l
 * `read_preference` - Specifies the read preference to be used by BI Connector for Atlas on the cluster. Each BI Connector for Atlas read preference contains a distinct combination of [readPreference](https://docs.mongodb.com/manual/core/read-preference/) and [readPreferenceTags](https://docs.mongodb.com/manual/core/read-preference/#tag-sets) options. For details on BI Connector for Atlas read preferences, refer to the [BI Connector Read Preferences Table](https://docs.atlas.mongodb.com/tutorial/create-global-writes-cluster/#bic-read-preferences).
 
+### Tags
+
+ Key-value pairs used for tagging and categorizing the cluster. Each key and value has a maximum length of 255 characters.
```

Comment:
I don't see any check here for limit 255 in key & values, shouldn't we do it here or it's ok if it's only done in the SDK?

Reply:
Within the terraform provider we currently are not defining any validations, such as regex, min/max lengths, or enums. We could analyse what benefit benefit it brings to the user taking into account the additional maintenance we add to the provider.